### DISCUSSION TITLE
Missing services?

### DISCUSSION THREAD

**Discussion by chasebolt** [humbertogontijo/homeassistant-roborock#436] [General]
I have a Q Revo and I do not have the service `roborock.vacuum_set_mop_mode`. Not sure what I am missing to have this service not listed. 
![image](https://github.com/humbertogontijo/homeassistant-roborock/assets/1222984/36740b35-01ab-45a6-83d3-b21950c9608e)

**Comment by humbertogontijo** [comment]
Use the service for the select entity instead

**Reply by chasebolt** [reply]
sorry but i dont follow. how do i use the service for the select entity? can i not use the developer tools > services?

**Reply by Lash-L** [reply]
@chasebolt its in that park as well.

Every entity type in HA comes with their own services.


So if you go to services and then put in select, you should see select: next select:first select: select

And you enter in what entity(as long as it is a select) you want to use that service for and then you put in what option you want selected ![image](https://github.com/humbertogontijo/homeassistant-roborock/assets/20257911/9c12f87e-4e72-4d46-bb3b-8c0238d5749c)

**Reply by chasebolt** [reply]
thanks. i understand now, i've never seen this before so i appreciate the image!
